Dockerfile

```diff
@@ -4,14 +4,16 @@ ARG PYTHON_VERSION=3.8
 # base image, build command: 
 # docker build --platform x86_64 --target towhee-base  -t towhee/towhee-base:latest .
 FROM ${BASE_IMAGE} as towhee-base
-RUN sed -i "s@http://.*archive.ubuntu.com@http://mirrors.tuna.tsinghua.edu.cn@g" /etc/apt/sources.list
-RUN sed -i "s@http://.*security.ubuntu.com@http://mirrors.tuna.tsinghua.edu.cn@g" /etc/apt/sources.list
+
+# Speed ​​up for Chinese users, only if user specified `USE_MIRROR=true`
+# In addition, use the default software source of the base image and language
+ARG USE_MIRROR=false
+ENV USE_MIRROR=${USE_MIRROR}
+RUN if [ "$USE_MIRROR" = "true" ]; then sed -i -e "s/archive.ubuntu.com/mirrors.tuna.tsinghua.edu.cn/" /etc/apt/sources.list && sed -i -e "s/security.ubuntu.com/mirrors.tuna.tsinghua.edu.cn/" /etc/apt/sources.list; fi
```

Comment:
we should add an argument 'mirror=', rather than giving a fixed mirro address

Reply:
and a list of mirro should be given in user docs

Reply:
Received suggestions, I think your idea is more suitable for users. @GuoRentong 

I will improve this commit.